hoomd/md/force.py

```diff
@@ -303,11 +291,10 @@ class Active(Force):
             It is defined per particle type and stays constant during
             the simulation.
         active_torque (tuple): active torque vector in reference to the
-            orientation of a particle
-            :math:`[\mathrm{force} \cdot \mathrm{length}]`.
-            It is defined per particle type and stays constant
-            during the simulation.
-
+            orientation of a particle. It is defined per particle type and stays
```

Comment:
```suggestion
            orientation of a particle :math:`[\mathrm{force} \cdot \mathrm{length}]`. It is defined per particle type and stays
```
Define units of the `active_torque` parameter. The text in this section will need to be wrapped appropriately after making this change.